src/main/webapp/script.js

```diff
@@ -70,6 +74,12 @@ function getCoordsFromSearch(geocoder, map) {
   });
 }
 
+// Create a Youtube player when api is loaded
```

Comment:
I'm not 100% sure, but you might want the comments above each function to be formatted with 
/**
 *
 */